testnet/Dockerfile.debug

```diff
@@ -0,0 +1,46 @@
+# BUILD IMAGE ---------------------------------------------------------
+
+FROM rust:1.85.1-slim-bookworm AS builder
+
+# Install dependencies needed for building RocksDB and etcd.
+RUN apt-get update && apt-get install -yq \
+    git gcc g++ clang libssl-dev pkg-config ca-certificates curl cmake ninja-build
+
+COPY . /nomos
+
+WORKDIR /opt
+
+RUN git clone https://github.com/risc0/risc0.git && \
+    cd risc0 && \
+    cargo install --path rzup && \
+    rzup build rust --tag-or-commit 4d91de4e48198da2e33413efdcd9cd2cc0c46688 && \
```

Comment:
Can you add a comment on what version does this hash correspond to?

Reply:
Added at https://github.com/logos-co/nomos/pull/1212/commits/0b6eea569b8bf8f87324c707a504ea1215e56e79